lib/util/index.ts

```diff
@@ -96,9 +70,6 @@ export const treatAsMissingDevice = (uuidOrId: string | number) => (
 	throw replacementErr;
 };
 
-export const isDevelopmentVersion = (version: string) =>
```

Comment:
It was only used in os.ts, so I just moved it there, since it might also make sense to export this method at some point.